src/test/java/org/opensearch/security/auth/InternalUserTokenHandlerTests.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.auth;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.identity.tokens.AuthToken;
+import org.opensearch.identity.tokens.BasicAuthToken;
+import org.opensearch.security.securityconf.impl.SecurityDynamicConfiguration;
+import org.opensearch.security.securityconf.impl.v7.InternalUserV7;
+import org.opensearch.security.user.InternalUserTokenHandler;
+import org.opensearch.security.user.UserService;
+import org.opensearch.security.user.UserServiceException;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.opensearch.security.dlic.rest.support.Utils.universalHash;
+import static org.opensearch.test.OpenSearchTestCase.assertEquals;
+
+
+public class InternalUserTokenHandlerTests  {
+    private InternalUserTokenHandler internalUserTokenHandler;
+    private UserService userService;
+    private SecurityDynamicConfiguration internalUsersConfiguration;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
+        userService = mock(UserService.class);
+        internalUsersConfiguration = mock(SecurityDynamicConfiguration.class);
+        when(userService.geInternalUsersConfigurationRepository()).thenReturn(internalUsersConfiguration);
+        internalUserTokenHandler = new InternalUserTokenHandler(Settings.EMPTY, userService);
+        internalUserTokenHandler.internalUsersConfiguration = internalUsersConfiguration;
+    }
+
+    @Test
+    public void testIssueTokenWithValidAccount() throws IOException {
+        when(userService.generateAuthToken("test")).thenReturn("Basic dGVzdDp0ZTpzdA=="); // test:te:st
```

Comment:
Just for my understanding what is meaning of this comment? Is this like the ${username}:${password}?

Reply:
Oh nvm I think I saw that at the below lines. :D